test/test.js

```diff
@@ -40,6 +59,35 @@ test('Passes parameters to the circuit function', (t) => {
     .catch(t.fail);
 });
 
+test('Using cache', (t) => {
+  t.plan(3);
+  const expected = 34;
+  const options = {
+    cache: true
+  };
+  const breaker = cb(passFail, options);
+
+  breaker.fire(expected)
+    .then((arg) => t.equals(arg, expected, `cache hits:misses ${breaker.status.cacheHits}:${breaker.status.cacheMisses}`))
+    .catch(t.fail)
+    .then(() => {
+      breaker.fire(expected)
+        .then((arg) => {
+          t.equals(arg, expected, `cache hits:misses ${breaker.status.cacheHits}:${breaker.status.cacheMisses}`);
+          breaker.clearCache();
+        })
+        .catch(t.fail)
+        .then(() => {
+          breaker.fire(expected)
+            .then((arg) => {
+              t.equals(arg, expected, `cache hits:misses ${breaker.status.cacheHits}:${breaker.status.cacheMisses}`);
```

Comment:
Similar to above, there should be a check here for 1 cache hit and 1 cache miss.